sdk/core/azure-mgmt-core/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.4.0 (Unreleased)
+## 1.4.0 (2023-04-06)
```

Comment:
just wanting to check, this is supposed to be empty ?

Reply:
Sorry. What is empty?

Reply:
no changes in the mgmt-core changelog ?

Reply:
The changes are already listed and non-needed bullets are already deleted. We only need to update release date. :)